### ISSUE TITLE
misleading x-axis

### ISSUE THREAD

**Issue by lldelisle** [deeptools/pyGenomeTracks#333]
```
$ pgt --version
pgt 3.6
$ pgt --tracks x_axis.ini --region chr1:0-19000 -o x_axis.png
```
With:
```
[x-axis]
```
Gives:
![x_axis](https://user-images.githubusercontent.com/30404086/113660279-05244c80-96a4-11eb-8964-f96c8ad35912.png)

We expect to have tick values = 0, 2.5, 5, 7.5... and we have 0, 2, 5, 8

**Comment by lldelisle** [comment]
It is now fixed in #343

**Comment by lldelisle** [comment]
Solved in version 3.7
